react-components/src/components/RevealToolbar/ResetCameraButton.tsx

```diff
@@ -20,13 +19,15 @@ export const ResetCameraButton = ({
 }: ResetCameraButtonProps): ReactElement => {
   const { t } = useTranslation();
   const cameraNavigation = useCameraNavigation();
+  const resetToDefault = useSceneDefaultCamera(sceneExternalId ?? '', sceneSpaceId ?? '');
```

Comment:
Can we rename this variable? Or instead write `const { fitCameraToSceneDefault } = useSceneDe....`?